src/controllers/notesController.js

```diff
@@ -0,0 +1,103 @@
+import mongooseModel from "../db/schemas/notesSchema.js";
+
+const notesController = {
+  get_all: async (req, res) => {
```

Comment:
Could you please change this strategy to classes or exported functions instead:
`export const getAllNotes`

And to reuse it inside other files: 
`import {getAllNotes} ...` or `import * as notesController ...`

Don't forget about clear naming and camel case instead of `_`